platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -57,6 +57,15 @@ Zinc has been updated to https://github.com/sbt/zinc/releases/tag/v1.10.4[1.10.4
 To determine the location of the Mac OS X SDK for Swift, Gradle now passes the `--sdk macosx` arguments to `xcrun`.
 This is necessary because the SDK could be discovered inconsistently without this argument across different environments.
 
+==== Source level deprecation of TaskContainer.create methods
+
+Eager task creation methods on the `TaskContainer` interface have been marked `@Deprecated` and will generate compiler and IDE warnings when used in build scripts or plugin code.
+There is not yet a Gradle deprecation warning emitted for their use.
+
+However, for builds that have configured to fail on warnings - either Kotlin script or plugin code compilation - this may cause the build to fail.
```

Comment:
```suggestion
However, for builds that have been configured to fail on warnings during either Kotlin script or plugin code compilation, this may cause the build to fail.
```